CHANGELOG.md

```diff
@@ -346,7 +347,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 -   Implement `DefinitionList` core component
 -   Implement `TransactionDataListItem.Skeleton` module component
--   Implement `OdsCoreProvider`, `LinkBase` and `AvatarBase` core components to support `Link` and `Image` NextJs
+-   Implement `GukCoreProvider`, `LinkBase` and `AvatarBase` core components to support `Link` and `Image` NextJs
```

Comment:
I would keep these names here, on previous versions the component will still be called `OdsXYZ`. Instead we can add a new entry under `unreleased` as `Rename x component to y`.
Let's specify on the changelog that we're renaming these components and the css custom variables